### ISSUE TITLE
Release mlfit 0.5.2

### ISSUE THREAD

**Issue by asiripanich** [mlfit/mlfit#62]
First release:

* [x] `usethis::use_cran_comments()`
* [x] Proof read `Title:` and `Description:`
* [x] Check that all exported functions have `@returns` and `@examples`
* [x] Check that `Authors@R:` includes a copyright holder (role 'cph') 90fa487
* [x] Check [licensing of included files](https://r-pkgs.org/license.html#code-you-bundle) 56cef0d 
* [x] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [x] `devtools::build_readme()`
* [x] [`urlchecker::url_check()`](https://github.com/r-lib/urlchecker)
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`

Submit to CRAN:

* [x] `usethis::use_version('patch')`
* [x] #65
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version()`
* [ ] Update install instructions in README

**Comment by asiripanich** [comment]
@krlmlr, I'm adding 'cph' role to your name.

**Comment by asiripanich** [comment]
Fixed the error as per CRAN's suggestion af7113a8f8ff6e0070b5566e168de7e8f14fa234

**Comment by asiripanich** [comment]
Comments from CRAN:
```
If there are references describing the methods in your package, please
add these in the description field of your DESCRIPTION file in the form
authors (year) <doi:...>
authors (year) <arXiv:...>
authors (year, ISBN:...)
or if those are not available: <https:...>
with no space after 'doi:', 'arXiv:', 'https:' and angle brackets for
auto-linking.
(If you want to add a title as well please put it in quotes: "Title")

Please add \value to .Rd files regarding exported methods and explain
the functions results in the documentation. Please write about the
structure of the output (class) and also what the output means. (If a
function does not return a value, please document that too, e.g.
\value{No return value, called for side effects} or similar)
Missing Rd-tags:
      compute_margins.Rd: \value
      fitting_problem.Rd:  \value


Please always add all authors, contributors and copyright holders in the
Authors@R field with the appropriate roles.
 From CRAN policies you agreed to:
"[....] Where copyrights are held by an entity other than the package
authors, this should preferably be indicated via ‘cph’ roles in the
‘Authors@R’ field, or using a ‘Copyright’ field (if necessary referring
to an inst/COPYRIGHTS file)."
e.g.: Andreas Alfons
Please explain in the submission comments what you did about this issue.

```
